app-emulation/xen-tools/xen-tools-4.19.1.ebuild

```diff
@@ -405,6 +409,10 @@ src_prepare() {
 			tools/firmware/ovmf-dir-remote/BaseTools/Source/C/VfrCompile/Pccts/*/makefile || die
 	fi
 
+	if ! use system-seabios ; then
+		sed -i "/^export HOSTCC/i override CC:=gcc" tools/firmware/seabios-dir/Makefile
```

Comment:
This is missing a die, which I'll add. External commands always need it if they may fail.